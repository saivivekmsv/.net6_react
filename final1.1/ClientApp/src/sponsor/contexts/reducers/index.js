export * from "./app-layout";
export * from "./manage-company";
export * from "./create-plan";
export * from "./manage-eligibility";
export * from "./manage-census";
export * from "./manage-payroll";
export * from "./manage-maintenance";
export * from "./manage-report";
export * from "./manage-mapper";
