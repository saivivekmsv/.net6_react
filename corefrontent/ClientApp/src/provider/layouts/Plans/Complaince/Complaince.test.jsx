import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";
import Complaince from "./Complaince";
import { CreatePlanProvider } from "../../../contexts";

test("Complaince", () => {
  const wrapper = render(
    <CreatePlanProvider>
      <Router>
        <Complaince />
      </Router>
    </CreatePlanProvider>
  );
  expect(wrapper.baseElement).toMatchSnapshot();
});
