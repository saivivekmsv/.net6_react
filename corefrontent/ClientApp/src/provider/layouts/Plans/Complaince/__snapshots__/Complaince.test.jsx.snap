// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Complaince 1`] = `
<body>
  <div>
    <form
      class="h-100"
    >
      <div
        class="d-flex flex-column manage-company h-100"
      >
        <div
          class="d-flex layout-header align-items-center"
        >
          <div>
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="breadcrumb"
              >
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/home"
                    >
                      Home
                    </a>
                  </a>
                </li>
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/plans"
                    >
                      Manage Plan
                    </a>
                  </a>
                </li>
              </ol>
            </nav>
            <h5 />
          </div>
          <div
            class="d-flex align-items-center justify-content-center tab-wrapper"
          />
          <div
            class="ml-auto"
          >
            <div>
              <a
                href="/manage-plan/complaince"
              >
                <button
                  class="btn btn-secondary"
                  flow="add,save"
                  link="/manage-plan/complaince"
                  type="button"
                >
                  Cancel
                </button>
              </a>
                 
              <button
                class="btn btn-primary"
                flow="add"
                type="submit"
              >
                Save
              </button>
                 
            </div>
          </div>
        </div>
        <div
          class="layout-content h-100 overflow-auto"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="required md form-group"
              >
                <label
                  class="form-label"
                >
                  Required Complaince testing type
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    aria-label="Required Complaince testing type"
                    class="btn-group"
                    name="complainceTestingType"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="ACP Testing"
                      type="button"
                    >
                      ACP Testing
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="ADP Testing"
                      type="button"
                    >
                      ADP Testing
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="Coverage"
                      type="button"
                    >
                      Coverage
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="Top heavy"
                      type="button"
                    >
                      Top heavy
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="required sm form-group"
              >
                <label
                  class="form-label"
                >
                  Testing Method
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    aria-label="Testing Method"
                    class="btn-group"
                    name="testingMethod"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="Current Year"
                      type="button"
                    >
                      Current Year
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="Prior Year"
                      type="button"
                    >
                      Prior Year
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="required sm form-group"
              >
                <label
                  class="form-label"
                >
                  HCE defined
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    aria-label="HCE defined"
                    class="btn-group"
                    name="HCE"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="true"
                      type="button"
                    >
                      Yes
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="false"
                      type="button"
                    >
                      No
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="required sm form-group"
              >
                <label
                  class="form-label"
                >
                  QMAC
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    aria-label="QMAC"
                    class="btn-group"
                    name="QMAC"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="true"
                      type="button"
                    >
                      Yes
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="false"
                      type="button"
                    >
                      No
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="required sm form-group"
              >
                <label
                  class="form-label"
                >
                  QNEC
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    aria-label="QNEC"
                    class="btn-group"
                    name="QNEC"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="true"
                      type="button"
                    >
                      Yes
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="false"
                      type="button"
                    >
                      No
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="required sm form-group"
              >
                <label
                  class="form-label"
                >
                  Safe Harbour
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    aria-label="Safe Harbour"
                    class="btn-group"
                    name="safeHarbour"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="true"
                      type="button"
                    >
                      Yes
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="false"
                      type="button"
                    >
                      No
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-2"
            >
              <div
                class="required sm form-group"
              >
                <label
                  class="form-label"
                >
                  Filing due date for Form 5500
                   
                </label>
                <button
                  class="form-control dropSide  dropdown"
                  style="position: relative; display: flex; align-items: center;"
                  type="button"
                >
                  <select
                    style="visibility: hidden;"
                  >
                    <option
                      value=""
                    >
                      ------select-----
                    </option>
                  </select>
                  <div
                    class="dropSide-custom-select"
                  >
                    Month
                  </div>
                  <span
                    class="dropSide-custom-select-arrow"
                    style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                  >
                    <span
                      style="transform: rotate(90deg);"
                    >
                      &gt;
                    </span>
                  </span>
                </button>
              </div>
            </div>
            <div
              class="col-md-2"
            >
              <div
                class=" xs form-group"
              >
                <label
                  class="form-label"
                >
                   
                </label>
                <button
                  class="form-control dropSide  dropdown"
                  style="position: relative; display: flex; align-items: center;"
                  type="button"
                >
                  <select
                    style="visibility: hidden;"
                  >
                    <option
                      value=""
                    >
                      ------select-----
                    </option>
                  </select>
                  <div
                    class="dropSide-custom-select"
                  >
                    Date
                  </div>
                  <span
                    class="dropSide-custom-select-arrow"
                    style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                  >
                    <span
                      style="transform: rotate(90deg);"
                    >
                      &gt;
                    </span>
                  </span>
                </button>
              </div>
            </div>
          </div>
          <div
            class="mt-20 row"
          >
            <div
              class="col-md-2"
            >
              <p
                class="form-label-text"
              >
                Summary Plan Description
              </p>
            </div>
            <div
              class="col-md-1"
            >
              <button
                class="btn-primary-view"
              >
                VIEW
              </button>
            </div>
          </div>
          <div
            class="mt-20 row"
          >
            <div
              class="col-md-2"
            >
              <p
                class="form-label-text"
              >
                Plan Legal Document 
              </p>
            </div>
            <div
              class="col-md-1"
            >
              <button
                class="btn-primary-view"
              >
                VIEW
              </button>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;
